tests/components/owntracks/test_device_tracker.py

```diff
@@ -622,9 +622,10 @@ async def test_event_gps_entry_exit_wrong_order(hass, context):
 async def test_event_gps_entry_unknown_zone(hass, context):
     """Test the event for unknown zone."""
     # Just treat as location update
-    message = build_message({"desc": "unknown"}, REGION_GPS_ENTER_MESSAGE)
+    lat = INNER_ZONE["latitude"]
```

Comment:
Modified test because with new zone criteria the enter message does not make us enter `inner` zone because the accuracy is too poor.